packages/kcmsf/src/hooks/useCheckData.ts

```diff
@@ -0,0 +1,65 @@
+import { isDepartmentType, isRobotType } from "config";
+import { CSVRow } from "../pages/registerBulk";
+import { errorMessages } from "../utils/notifyError";
+
+export type ErrorData = {
+  teamName: string[];
+  member1: string[];
+  member2: string[];
+  robotType: string[];
+  departmentType: string[];
+  clubName: string[];
+};
```

Comment:
今のところ1つのセルに対して2つ以上のエラーを出すケースはなさそうなので、配列である必要はないように思います。
```suggestion
export type ErrorData = {
  teamName?: string;
  member1?: string;
  member2:? string;
  robotType?: string;
  departmentType?: string;
  clubName?: string;
};
```